### PR TITLE
Update data loader with course average rating

### PR DESCRIPTION
**Pull request by etechdevai**
**Description:**  This PR updates data loader and course search and course run APIs with course average rating

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3767

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.

### CODE REVIEW

**Review by rehan99000 — APPROVED** [review]

**Review by taimoor-ahmed-1 — APPROVED** [review]

File: course_discovery/apps/course_metadata/search_indexes.py [hunk 1]
```diff
@@ -250,6 +250,8 @@ class CourseRunIndex(BaseCourseIndex, indexes.Indexable):
     marketing_price_value = indexes.CharField(model_attr='marketing_price_value', null=True)
     is_marketing_price_hidden = indexes.BooleanField(model_attr='is_marketing_price_hidden', null=True)
     card_image_url = indexes.CharField(model_attr='card_image_url', null=True)
+    average_rating = indexes.DecimalField(model_attr='average_rating', null=True)
```
Comment (burhandodhy) [inline course_discovery/apps/course_metadata/search_indexes.py:253]: Could you please double check that sorting is working fine?

Comment (etechdevai) [reply course_discovery/apps/course_metadata/search_indexes.py:253]: Yes, sorting/ordering is working. I have verified it on DEV:
https://discovery.multisitesdev.edly.io/api/v1/course_runs/?limit=50&offset=0&ordering=-total_raters

Comment (burhandodhy) [reply course_discovery/apps/course_metadata/search_indexes.py:253]: @asadmanzoor93 please check on the `search` endpoint.

Comment (etechdevai) [reply course_discovery/apps/course_metadata/search_indexes.py:253]: @burhandodhy working fine with search endpoint as well:
https://discovery.multisitesdev.edly.io/api/v1/search/course_runs/?ordering=-total_raters

**Review by burhandodhy — APPROVED** [review]

**Pull request merged**